Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shipyard.yml

```diff
@@ -121,6 +134,9 @@
     interfaces:
       enum.ShipyardConsoleUiKey.Security:
         type: ShipyardConsoleBoundUserInterface
+  - type: ShipyardConsole
+    shipyardChannel: Nfsd
+    newAccessLevels: [Captain, Security, Brig]
```

Comment:
Might be dumb, but is this defining new behaviour or inheriting from what already exists?  Seems sane, only issue I could imagine is if a prisoner or other antagonist was able to buy a ship, not sure if that's possible OTOH.

Reply:
Inheriting existing behaviour:

https://github.com/new-frontiers-14/frontier-station-14/blob/e77ade7db7d8d0a9c68f865457d773990689eb9f/Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs#L178-L182